v2.0/app/containers/MassEnergizeSuperAdmin/Pages/AboutUs.js

```diff
@@ -1,34 +1,33 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import { withStyles } from '@material-ui/core/styles';
-import MassEnergizeForm from '../_FormGenerator';
-import { apiCall } from '../../../utils/messenger';
+import React, { Component } from "react";
```

Comment:
Like the other case, please don't commit files with hundreds of formatting changes.  Drives me crazy